README.md

```diff
@@ -13,10 +13,13 @@ Oculus Legacy  (Linux, Mac, Windows)
 Oculus (Windows)
 * Oculus 0.7 runtime
 
-Note
-====
-* Windows 64 builds are not working at the moment, use 32 bits instead
-* Extended Mode is the only supported mode
+HTC Vive (Windows)
+* OpenVR 0.9.20 runtime
+
+Current Known Issues
+====================
+HTC Vive: TODO: Fix Blender crash on OpenVR when stopping. (Pausing works).
```

Comment:
Any clues on why you are getting this crash? Do you have a segfault or it isreally simply a freeze?